envoy/server/factory_context.h

```diff
@@ -288,6 +288,13 @@ class FilterChainFactoryContext : public virtual FactoryContext {
 };
 
 using FilterChainFactoryContextPtr = std::unique_ptr<FilterChainFactoryContext>;
+using FilterChainsByName = absl::flat_hash_map<std::string, Network::DrainableFilterChainSharedPtr>;
+
+class FilterChainBaseAction : public Matcher::Action {
```

Comment:
in retrospect I should have commented this up when moving it.  If there end up being no other comments I'm inclined to do in the immediate follow-up to spare the 4 hour CI process.

Reply:
SGTM! I did have one additional comment but it was to add a comment, so feel free to address both in a follow up instead of waiting on CI again!